### DISCUSSION TITLE
React Hook Form and Draft JS - when I submit the form, DraftJS values are not registered.

### DISCUSSION THREAD

**Discussion by lukebyrne73** [react-hook-form/react-hook-form#2388] [Q&A]
I'm using DraftJS and React Hook Form to have a rich text editor. When I try to use a custom controller, the input on the controller does not register with my form. I want to know if there's something that I'm missing here. 

[CodeSandbox](https://codesandbox.io/s/amazing-kapitsa-2fwjd)

Thank you, I'm a big fan of React Hook Form!

**Comment by bluebill1049** [comment]
I think i will need to do a draftJS example, as many people request for the same question.

**Reply by bluebill1049** [reply]
i have updated the codesandbox in the doc: https://codesandbox.io/s/react-hook-form-v6-controller-qsd8r?file=/src/ButtonsResult.js let me know if this helps.

**Reply by lukebyrne73** [reply]
Hi Bill, the version you put up is helpful (thank you!) but I'm still getting errors. My codesandbox has been updated and I'm getting a new error: 
https://codesandbox.io/s/amazing-kapitsa-2fwjd

**Reply by bluebill1049** [reply]
can you do an isolated example? more than happy to help but my time is limited due to work, issues and etc.. see if you replicate the issue at ioslate codesandbox and then work backwards.
